cli/txctx/tx.go

```diff
@@ -68,12 +77,32 @@ func SignAndSend(ctx *cli.Context, act *actor.Actor, acc *wallet.Account, tx *tr
 			// Compensate for confirmation waiting.
 			tx.ValidUntilBlock += uint32((waitTime.Milliseconds() / int64(ver.Protocol.MillisecondsPerBlock))) + 1
 		}
-		_, _, err = act.SignAndSend(tx)
+		resTx, vub, err := act.SignAndSend(tx)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
+
+		if ctx.Bool("await") {
+			aer, err = act.Wait(resTx, vub, err)
+		}
```

Comment:
Linter tells "ineffectual assignment to err at L86. Can be fixed with:
```suggestion
		var (
			resTx util.Uint256
			vub int64
		)
		resTx, vub, err = act.SignAndSend(tx)
		if err != nil {
			return cli.NewExitError(err, 1)
		}

		if ctx.Bool("await") {
			aer, err = act.Wait(resTx, vub, err)
		}
```